app-text/namazu/namazu-2.0.22_pre8.ebuild

```diff
@@ -10,10 +10,10 @@ MY_P="${P/_pre/pre}"
 DESCRIPTION="Namazu is a full-text search engine"
 HOMEPAGE="http://www.namazu.org/"
 SRC_URI="http://www.namazu.org/test/${MY_P}.tar.gz"
+S="${WORKDIR}"/${MY_P}
 
 LICENSE="GPL-2+"
 SLOT="0"
-KEYWORDS=""
```

Comment:
Hmm, while this indeed prints a pkgcheck scan warning, removing this line makes using tools such as **ekeyword** impossible. I wonder what we should do here, I'll probably file a pkgcheck issue for it.

Reply:
NAK.

Reply:
It is a super minor notice not even rising to the level of a warning, just ignore it.